From 12e0534dad3a609d35004784a57d5a1cd74f71d7 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Thu, 4 Aug 2011 10:33:27 +0800
Subject: [PATCH 18/38] ENGR00154297 Enlarge the wait time for ppp connection

the time for ppp connection setup may more than 15 seconds for china mobile

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 reference-ril/reference-ril.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/reference-ril/reference-ril.c b/reference-ril/reference-ril.c
index 4780fa1..3d83150 100755
--- a/reference-ril/reference-ril.c
+++ b/reference-ril/reference-ril.c
@@ -57,8 +57,9 @@
 #define PPP_OPERSTATE_PATH "/sys/class/net/ppp0/operstate"
 #define SERVICE_PPPD_GPRS "pppd_gprs"
 #define PROPERTY_PPPD_EXIT_CODE "net.gprs.ppp-exit"
-#define POLL_PPP_SYSFS_SECONDS	3
-#define POLL_PPP_SYSFS_RETRY	3
+// Max wait time to 2*10 secondes for ppp enable
+#define POLL_PPP_SYSFS_SECONDS 2
+#define POLL_PPP_SYSFS_RETRY   10
 
 #ifdef USE_TI_COMMANDS
 
-- 
1.8.0

